css/style.css

```diff
@@ -0,0 +1,173 @@
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  box-sizing: border-box;
+}
+
+.profile {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 40px;
+}
+
+.profile-picture {
+  width: 150px;
+  height: 150px;
+  border-radius: 16px;
+}
+
+.profile-name {
+  margin-top: 10px;
+  margin-bottom: 5px;
+  font-size: 24px;
+  color: #4e5150;
+}
+
+.profile-place {
+  margin-top: 5px;
+  margin-bottom: 10px;
+  font-size: 14px;
+  color: #a9a9a9;
+}
+
+.profile-socials {
+  display: flex;
+  gap: 24px;
+}
+
+.profile-socials-item {
+  width: 100px;
+  height: 67px;
+  border-radius: 12px;
+  background-color: #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.count {
+  font-size: 18px;
+  margin-bottom: 5px;
+  color: #4e5150;
+}
+
+.count-text {
+  font-size: 14px;
+  margin-top: 0px;
+  color: #bdbdbd;
+}
+
+.image-1 {
+  grid-area: image1;
```

Comment:
In this case I believe we can do it without grid-area, because it'll be shorter and won't lose readability